src/test/ui/lint/lint-temporary-cstring-as-param.rs

```diff
@@ -0,0 +1,10 @@
+// ignore-tidy-linelength
+#![deny(temporary_cstring_as_ptr)]
+
+use std::ffi::CString;
+
+fn some_function(data: *const i8) {}
+
+fn main() {
+    some_function(CString::new("").unwrap().as_ptr()); //~ ERROR getting the inner pointer of a temporary `CString`
```

Comment:
Why not just remove the ignore tidy linelength check and reduce this line length?
```suggestion
    some_function(CString::new("").unwrap().as_ptr());
    //~^ ERROR getting the inner pointer of a temporary `CString`
```